apps/dashboards/tests/test_api.py

```diff
@@ -45,15 +45,26 @@ def test_list_view(self):
         response = self.client.get(url)
         assert response.status_code == 200
 
+    def test_list_view_as_staff(self):
+        request = self.factory.get(
+            reverse("visualization-list"),
+        )
+        request.user = self.user
+        request.user.is_staff = True
+        view = VisualizationViewSet.as_view(actions={"get": "list"})
+        response = view(request)
+        assert response.data["count"] > 0
+        assert response.status_code == 200
+
     def test_list_view_as_non_staff(self):
         request = self.factory.get(
             reverse("visualization-list"),
         )
         request.user = self.user
         request.user.is_staff = False
         view = VisualizationViewSet.as_view(actions={"get": "list"})
-        force_authenticate(request, user=self.user)
         response = view(request)
+        assert response.data["count"] >= 0
```

Comment:
Same as above, specify the count. Show that there is an actual difference between the number of visualizations that a staff user can access as compared to a non- staff user